templates/auth0-sso-handler-lock10.php

```diff
@@ -12,10 +12,14 @@
     domain:'<?php echo $domain; ?>'
   });
 
-  webAuth.client.getSSOData(function(err, data) {
-    if (!err && data.sso) {
-      webAuth.authorize(<?php echo json_encode( $lock_options->get_sso_options() ); ?>);
-    }
-  });
+  var options = <?php echo json_encode( $lock_options->get_sso_options() ); ?>;
+  webAuth.checkSession(options
+  , function (err, authResult) {
+      if (typeof(authResult) === 'undefined') {
+        return;
+      }
+      window.location = '<?php echo home_url( '/?auth0=1' ); ?>&code=' + authResult.code + '&state=' + authResult.state;
+    });
```

Comment:
what if it is using the implicit flow?
in that case you might need to do the same than the login screen (post back to wordpress).

Or maybe we can use `response_mode=form_post` and in this case in particular make it go to the login callback and after that refresh the user agent.